etcdmain/config.go

```diff
@@ -258,6 +258,7 @@ func newConfig() *config {
 	fs.IntVar(&cfg.ec.ExperimentalCompactionBatchLimit, "experimental-compaction-batch-limit", cfg.ec.ExperimentalCompactionBatchLimit, "Sets the maximum revisions deleted in each compaction batch.")
 
 	// unsafe
+	fs.BoolVar(&cfg.ec.UnsafeNoFsync, "unsafe-no-fsync", false, "Disables fsync, unsafe, will cause data loss.")
```

Comment:
Can we please mark it as hidden flag?

Reply:
Looks like it already is, although I'm not sure that is what we had intended for master:

https://github.com/etcd-io/etcd/blob/49f91d629a78c3d6ae285aec7f4bb3b00fb49c03/etcdmain/help.go#L26-L31

... no `unsafe-no-fsync` section

Reply:
excellent. thanks @jpbetz